CONTRIBUTING.md

```diff
@@ -52,4 +52,9 @@ If the project maintainer has any additional requirements, you will find them li
 
 - **Send coherent history** - Make sure each individual commit in your pull request is meaningful. If you had to make multiple intermediate commits while developing, please [squash them](https://www.git-scm.com/book/en/v2/Git-Tools-Rewriting-History#Changing-Multiple-Commit-Messages) before submitting.
 
-**Happy coding**!
\ No newline at end of file
+**Happy coding**!

```

Comment:
```suggestion
Happy coding!

```

Bold formatting above makes sense to distinguish the text from the remainder but I don't think it adds anything here. Let me know if this is enough bike-shedding for you. I can go deeper.

Reply:
Done - I didn't want to bother to update the contribution docs too much here, but I believe a second pass in the near future is in order.